scalgrafanalib/core.py

```diff
@@ -1,9 +1,9 @@
-from typing import Any, Dict, List, TypeVar, Union
+from typing import Any, Dict, List, TypeVar
 import attr
 from grafanalib import core  # type: ignore
 
+Self = TypeVar("Self", bound="Dashboard")
```

Comment:
this should be defined as `Self = TypeVar("Self")` (as it was before), and not bound to a specific type...

this was working fine a few month back, so maybe we need to make this conditional on python version: `TypeVar("Self")` (i.e. original code) for older python, and typing.Self for newer ones... :-(

(or just require a newer python version, and drop support for older ones)

Reply:
I won't bump anything in this PR, we can do it separately.